### PR TITLE
kal-text-area : Creation of kal-textarea component

### PR DESCRIPTION
**Pull request by MichaelGrosio**

### CODE REVIEW

**Comment by ydomenjoud** [comment]
add playground entry

File: projects/kalidea/kaligraphi/src/lib/molecules/kal-textarea/kal-textarea.component.spec.ts [hunk 1]
```diff
@@ -1,14 +1,20 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { KalTextareaComponent } from './kal-textarea.component';
+import { ReactiveFormsModule } from '@angular/forms';
```
Comment (jsp-up) [inline projects/kalidea/kaligraphi/src/lib/molecules/kal-textarea/kal-textarea.component.spec.ts]: imports

File: projects/kalidea/kaligraphi/src/lib/molecules/kal-textarea/kal-textarea.component.ts [hunk 2]
```diff
@@ -9,13 +18,40 @@ import { buildProviders, FormElementComponent } from '../../utils/index';
...
+
   ngOnInit() {
+    this.subscription = this.formControl.valueChanges.subscribe(
+      value => {
+        this.notifyUpdate(value);
+      }
+    );
+  }
+
+  ngOnDestroy() {
```
Comment (jsp-up) [inline projects/kalidea/kaligraphi/src/lib/molecules/kal-textarea/kal-textarea.component.ts]: unsubscribe

**Review by jsp-up — CHANGES_REQUESTED** [review]

**Review by jsp-up — DISMISSED** [review]

**Review by ydomenjoud — APPROVED** [review]

**Comment by jsp-up** [comment]
Close #95

**Pull request merged**